Program.cs

```diff
@@ -1,7 +1,34 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System.Text;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter Password: ");
+         string Password = Console.ReadLine ();
+         (bool isStrong, string errorMsg) = checkPassword (Password);
+         if (isStrong) Console.WriteLine ("Strong password.");
+         else Console.WriteLine ("Password is not strong.\n" + errorMsg);
+      }
+      static (bool isStrong, string errorMsg) checkPassword (string pass) {
+         StringBuilder err = new ();
+         bool lower = false, upper = false, digit = false, splChar = false;
+         foreach (char c in pass) {
+            if (char.IsLower (c))
+               lower = true;
+            if (char.IsUpper (c))
+               upper = true;
+            if (char.IsDigit (c))
+               digit = true;
+            string spl = "!@#$%^&*()-+";

```

Comment:
Repeated statement/assignment could be defined outside the scope.

Reply:
Changed